### DISCUSSION TITLE
how do i change any default snacks.nvim configuration in lazyvim?

### DISCUSSION THREAD

**Discussion by Nirajgns** [folke/snacks.nvim#40] [General]
i am unable to change anything related to snacks.nvim in my lazyvim config.

**Comment by f1uff3h** [comment]
Can you post what didn't work or how you tried changing snacks config? As with any other plugin you can configure via files in `~/.config/nvim/lua/plugins/yourconfig.lua` see [docs](https://www.lazyvim.org/configuration/plugins)

yourconfig.lua:
```lua
return {
  {
    "folke/snacks.nvim"
    <your-configs-here>
  },
}
```
